### DISCUSSION TITLE
concordancer中 设置栏内的zapping settings是什么意思？

### DISCUSSION THREAD

**Discussion by Sam-zcl** [BLKSerene/Wordless#26] [Q&A] [answered]
![image](https://github.com/BLKSerene/Wordless/assets/44527431/14986a44-f90c-4b82-9111-a6722e823cd8)
想请问图中的zapping setting是什么意思？在文档中我似乎没有搜索到说明，谢谢！

**Comment by BLKSerene** [comment, marked as answer]
检索项遮蔽功能 勾选这个后导出为 Word 文档 Node 会被替换为下划线 就是类似于完形填空的样子
可以参考 B 站 2.2.0 的教程第 p4 的最后：https://www.bilibili.com/video/BV1VF41137zH?p=4&vd_source=7a820de41a926600601756d5178bf28a

**Comment by Sam-zcl** [comment]
thx! really appreciate it.
